sdk/src/main/scala/com/horizen/SidechainHistory.scala

```diff
@@ -491,7 +491,7 @@ object SidechainHistory
       SidechainBlockInfo.mainchainHeaderHashesFromBlock(block),
       SidechainBlockInfo.mainchainReferenceDataHeaderHashesFromBlock(block),
       WithdrawalEpochInfo(1, block.mainchainBlockReferencesData.size), // First Withdrawal epoch value. Note: maybe put to params?
-      new VrfProofHash(Array()), //vrfProofHash is not used at all, so use just placeholder
+      new VrfOutput(Array()), //vrfOutput is not used at all, so use just placeholder
```

Comment:
In case of genesis block - probably now `None` is better then `new VrfOutput(Array())`
To be discussed.